sormas-backend/src/main/java/de/symeda/sormas/backend/externaljournal/ExternalJournalService.java

```diff
@@ -220,14 +225,29 @@ public boolean notifyExternalJournalPersonUpdate(JournalPersonDto existingJourna
 		return shouldNotify;
 	}
 
+	public void handleExternalJournalPersonUpdate(PersonReferenceDto person) {
+		if (!configFacade.isExternalJournalActive()) {
+			return;
+		}
+
+		/**
+		 * The .getPersonForJournal(...) here gets the person in the state it is (most likely) known to an external journal.
+		 * Changes of related data is assumed to be not yet persisted in the database.
+		 * 5 second delay added before notifying of update so that current transaction can complete and
+		 * new data can be retrieved from DB
+		 */
+		JournalPersonDto existingPerson = personFacade.getPersonForJournal(person.getUuid());
+		Runnable notify = () -> notifyExternalJournalPersonUpdate(existingPerson);
```

Comment:
I would inline the Runnable notify, I see no value having it assigned.

Reply:
Done.